pkgs/nix-tools/darwin-rebuild.sh

```diff
@@ -126,7 +126,7 @@ if [ -n "$flake" ]; then
 fi
 
 if [ -n "$flake" ]; then
-    flake=$(nix "${flakeFlags[@]}" flake info --json "${extraBuildFlags[@]}" "${extraLockFlags[@]}" -- "$flake" | jq -r .url)
+    flake=$(nix "${flakeFlags[@]}" flake metadata --json "${extraBuildFlags[@]}" "${extraLockFlags[@]}" -- "$flake" | jq -r .url)
```

Comment:
Something kind of like this will probably work.

```suggestion
    if nix flake metadata --help >/dev/null; then
        cmd=metadata
    else
        cmd=info
    fi
    flake=$(nix "${flakeFlags[@]}" flake "$cmd" --json "${extraBuildFlags[@]}" "${extraLockFlags[@]}" -- "$flake" | jq 
```

Reply:
Alright. Pushed. I tested my fork and it looks fine.